stable/datacube/templates/wms-deployment.yaml

```diff
@@ -13,6 +13,19 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
+  securityContext:
+    sysctls:
+      # If this flag is set to 1 then all System V shared memory segments will be marked for destruction as soon as
+      # the number of attached processes falls to zero
+      - name: kernel.shm_rmid_forced
+        value: "0"
+      # comment out ipv6 localhost
+      - name: net.ipv6.conf.all.disable_ipv6
+        value: "1"
+      - name: net.ipv6.conf.default.disable_ipv6
+        value: "1"
+      - name: net.ipv6.conf.lo.disable_ipv6
+        value: "1"
```

Comment:
What are these changes for? They should be in a seperate PR

Reply:
Removed these changes. These changes were added as part of `gunicorn` `ulimit` updates. I will add them in another PR.